### DISCUSSION TITLE
Error When Trying to Push to GitHub from Terminal

### DISCUSSION THREAD

**Discussion by kayleybannister** [community/community#30727] [Other Feature Feedback, Questions, & Ideas]
Hey everyone! Trying to get back into coding after taking a break and I cannot get my push to go through to GitHub. Below is the error that shows up once I enter git push. I am sure it is something basic, just am stuck...

git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Comment by airtower-luna** [comment]
The error message indicates your SSH key wasn't accepted. Most likely that means either you don't have one, or the one you have isn't on your GitHub account.

You mentioned taking a break, depending on how long that was it's possible [it was deleted for inactivity](https://docs.github.com/en/authentication/troubleshooting-ssh/deleted-or-missing-ssh-keys). Either way the right approach is to [add your current SSH key to your account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account), after creating a new one if necessary.

If you more have SSH issues try running `ssh -v git@github.com` to get debug info on the connection.
